[remote] remote launcher plugin when trying to use jmx=attach from bndtools through ClassNotFoundException

```
java.lang.NoClassDefFoundError: javax/management/remote/JMXServiceURL
	at aQute.remote.util.JMXBundleDeployer.getLocalConnectorAddress(JMXBundleDeployer.java:283)
	at aQute.remote.util.JMXBundleDeployer.<init>(JMXBundleDeployer.java:31)
	at aQute.remote.plugin.RemoteProjectLauncherPlugin.tryJMXDeploy(RemoteProjectLauncherPlugin.java:225)
	at aQute.remote.plugin.RemoteProjectLauncherPlugin.prepare(RemoteProjectLauncherPlugin.java:114)
	at aQute.remote.plugin.RemoteProjectLauncherPlugin.getRunSessions(RemoteProjectLauncherPlugin.java:206)
	at bndtools.launch.bnd.NativeBndLaunchDelegate.launch(NativeBndLaunchDelegate.java:117)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:885)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:739)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1039)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$8.run(DebugUIPlugin.java:1256)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:54)
Caused by: java.lang.ClassNotFoundException: javax.management.ObjectName not found, parent:  org.eclipse.osgi.internal.loader.EquinoxClassLoader@4c6a3e4a[biz.aQute.bndlib:3.1.0.201511110446(id=4)] urls:[file:/Users/greg/dev/repos/bndtools/bndtools.test/bndws/cnf/cache/3.1.0/biz.aQute.remote.launcher/biz.aQute.remote.launcher-3.1.0.jar] exception:java.lang.ClassNotFoundException: javax.management.ObjectName
	at aQute.bnd.osgi.Processor$CL.loadClass(Processor.java:1621)
	... 12 more
Caused by: java.lang.ClassNotFoundException: javax.management.ObjectName
	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at aQute.bnd.osgi.Processor$CL.loadClass(Processor.java:1609)
	... 12 more
```

Here is the problem:
```
Caused by: java.lang.ClassNotFoundException: javax.management.ObjectName not found, parent:  org.eclipse.osgi.internal.loader.EquinoxClassLoader@4c6a3e4a[biz.aQute.bndlib:3.1.0.201511110446(id=4)] urls:[file:/Users/greg/dev/repos/bndtools/bndtools.test/bndws/cnf/cache/3.1.0/biz.aQute.remote.launcher/biz.aQute.remote.launcher-3.1.0.jar] exception:java.lang.ClassNotFoundException: javax.management.ObjectName
 
```
The bndlib parent bundle doesn't import any of the javax.management.* packages.  I hacked up a new bndlib bundle that had the proper imports and then this remote launcher error goes away.  Thoughts?

--- a/org.bndtools.embeddedrepo/bnd.bnd
+++ b/org.bndtools.embeddedrepo/bnd.bnd
@@ -3,3 +3,7 @@
 
 Fragment-Host: biz.aQute.bndlib;bundle-version="[${bnd-version-base},${bnd-version-ceiling})"
 -includeresource: embedded-repo.jar=${repo;biz.aQute.bnd.embedded-repo;[${bnd-version-base},${bnd-version-ceiling})}
+
+Import-Package: \
+	javax.management,\
+	javax.management.remote

